__init__.py

```diff
@@ -80,166 +105,40 @@ def _wiki_page_summary(self, result, auto_suggest):
             auto_suggest (bool): True if auto suggest was used to get this
                                  result.
         """
-        lines = 2
+        lines = 20
         summary = wiki.summary(result, lines, auto_suggest=auto_suggest)
 
-        if "==" in summary or len(summary) > 250:
-            # We hit the end of the article summary or hit a really long
-            # one.  Reduce to first line.
+        if "==" in summary or len(summary) > 2500:
+            # We hit the end of the article summary or hit a really 
+            # long one.  Reduce to first line.
+            # TODO - this could be better, correct this eventually
             lines = 1
             summary = wiki.summary(result, lines, auto_suggest=auto_suggest)
 
         # Clean text to make it more speakable
         return re.sub(r'\([^)]*\)|/[^/]*/', '', summary), lines
 
-    def serialize(self):
-        """Serialize the object to string.
-
-        Returns:
-            (str) string represenation of the object
-        """
-        return json.dumps(self.__dict__)
-
-    @classmethod
-    def deserialize(cls, data):
-        """Create a PageMatch object from serialized version."""
-        input_dict = json.loads(data)
-        return cls(result=input_dict['wiki_result'],
-                   auto_suggest=input_dict['auto_suggest'],
-                   summary=input_dict['summary'],
-                   lines=input_dict['lines'],
-                   image=input_dict['image']
-                   )
-
-
-def wiki_lookup(search, lang_code, auto_suggest=True):
-    """Performs a wikipedia article lookup.
-
-    Arguments:
-        search (str): phrase to search for
-        lang_code (str): wikipedia language code to use
-        auto_suggest (bool): wether or not to use autosuggest.
-
-    Returns:
-        PageMatch, PageDisambiguation or None
-    """
-    try:
-        # Use the version of Wikipedia appropriate to the request language
-        wiki.set_lang(lang_code)
-
-        # Fetch wiki article titles. This comes back
-        # as a list.  I.e. "beans" returns ['beans',
-        #     'Beans, Beans the Music Fruit', 'Phaseolus vulgaris',
-        #     'Baked beans', 'Navy beans']
-        results = wiki.search(search, 5)
-        if len(results) == 0:
-            return None
+class WikipediaSkill(CommonQuerySkill):
+    """TODO these should also be moved out to a voc file
+    so we have a chance at supporting something
+    besides english.
 
-        return PageMatch(results[0], auto_suggest)
+    Only ones that make sense in
+    <question_word> <question_verb> <noun>"""
+    question_words = ['who', 'whom', 'what', 'when']
+    # Note the spaces
+    question_verbs = [' is', '\'s', 's', ' are', '\'re',
+                      're', ' did', ' was', ' were']
+    articles = ['a', 'an', 'the', 'any']
```

Comment:
Can we make this translatable before merging? See [MycroftSkill.translate](https://mycroft-core.readthedocs.io/en/latest/source/mycroft.html#mycroft.MycroftSkill.translate)

Reply:
Yea, so this is the ago old question, when you modify a skill do you now become responsible for all its previous baggage. In most cases I say no but in this case I agree. I will pull all text out so the skill has a chance to be multi-lingual.

Reply:
Is there a preferred way to handle language translation? I was under the assumption that text placed in a voc file would suffice as ultimately these would be translated. But I see Ake reference MycroftSkill.translate which is new to me. Is this the preferred method or is a voc file good enough?

Reply:
You can use the voc's but I think it's trickier.